tests/unit/amo/components/TestSearchResult.js

```diff
@@ -144,6 +144,85 @@ describe(__filename, () => {
     expect(root).toHaveClassName('SearchResult--theme');
   });
 
+  it('displays the thumbnail image as the default src for static theme', () => {
+    const headerImageThumb =
+      'https://addons.cdn.mozilla.net/mytestthumb/12345.png';
+    const newPreview = [
```

Comment:
You don't really need this `newPreview` variable, nor the `addon` variable below - you could just inline these into the call to `render`.

Reply:
yep i'll do that :)